File: src/test/java/net/botlify/anticaptchacom/response/solution/TurnstileSolutionTest.java
package net.botlify.anticaptchacom.response.solution;

import com.fasterxml.jackson.databind.ObjectMapper;
import lombok.SneakyThrows;
import net.botlify.anticaptchacom.TestUtils;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class TurnstileSolutionTest {

  @Test
  @SneakyThrows
  public void testParseJson() {
    final String fileName = "response/customTaskResponse.json";
    final String fileContent = TestUtils.readResourceFile(fileName);

    final ObjectMapper mapper = new ObjectMapper();
    final CustomTaskResponse response = mapper.readValue(fileContent, CustomTaskResponse.class);
  }

}

File: src/test/kotlin/net/botlify/anticaptchacom/response/solution/TurnstileSolutionTest.kt
package net.botlify.anticaptchacom.response.solution

import com.fasterxml.jackson.databind.ObjectMapper
import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
import net.botlify.anticaptchacom.TestUtils
import net.botlify.anticaptchacom.data.response.solution.CustomTaskResponse
import org.junit.jupiter.api.Test
import kotlin.test.assertNotNull

internal class TurnstileSolutionTest {

    @Test
    fun testParseJson() {
        val fileName = "response/customTaskResponse.json"
        val fileContent: String = TestUtils.readResourceFile(fileName)

        val mapper = jacksonObjectMapper()
        val response: CustomTaskResponse = mapper.readValue(fileContent, CustomTaskResponse::class.java)
        assertNotNull(response)
    }
}